redash/query_runner/dremio.py

```diff
@@ -0,0 +1,119 @@
+from redash.query_runner import BaseQueryRunner, register
+from redash.query_runner import TYPE_STRING, TYPE_DATE, TYPE_DATETIME, TYPE_INTEGER, TYPE_FLOAT, TYPE_BOOLEAN
+from redash.utils import json_dumps, json_loads
+import pyodbc
+
+enabled = True
```

Comment:
You need to import this in a try/except block and set `enabled` accordingly. You can see examples for this pattern in other query runners.